Description: Fix scsitape usage message
Author: Carsten Leonhardt <leo@debian.org>
Last-Update: 2020-11-01

Add the "erase" operation to the usage message of scsitape. (Hint found at Rosa Linux)

--- a/scsitape.c
+++ b/scsitape.c
@@ -63,7 +63,7 @@
 #endif
 
 void Usage(void) {
-  FatalError("Usage: scsitape -f <generic-device> <command> where <command> is:\n setblk <n> | fsf <n> | bsf <n> | eod | rewind | eject | mark <n> |\n  seek <n> | read [<blksize> [<numblocks]] | write [<blocksize>] \n");
+  FatalError("Usage: scsitape -f <generic-device> <command> where <command> is:\n setblk <n> | fsf <n> | bsf <n> | eod | rewind | eject | erase |\n  mark <n> | seek <n> | read [<blksize> [<numblocks]] | write [<blocksize>]\n");
 }
 
 #define arg1 (arg[0])  /* for backward compatibility, sigh */
